data/src/main/java/com/globant/equattrocchio/data/ImagesServicesImpl.java

```diff
@@ -13,28 +14,28 @@
 
 public class ImagesServicesImpl implements ImagesServices {
 
-    private static final String URL= "http://splashbase.co/";
+    private static final String URL = "http://splashbase.co/";
 
     @Override
-    public void getLatestImages(Observer<Boolean> observer) {
+    public void getLatestImages(final Observer<String> observer) {
         Retrofit retrofit = new Retrofit.Builder().
                 baseUrl(URL).
                 addConverterFactory(GsonConverterFactory.create())
                 .build();
 
-        SplashbaseApi api  = retrofit.create(SplashbaseApi.class);
+        SplashbaseApi api = retrofit.create(SplashbaseApi.class);
```

Comment:
Refactor `SplashbaseApi` to `SplashBaseApi`. Always use camel case for class names.